src/main/features/response/views/confirmation/part-admission-steps.njk

```diff
@@ -79,8 +80,12 @@
   </ul>
 
   <h3 class="bold">{{ t('If {{ claimantName }} rejects that you only owe {{ amount }}', { claimantName: claimantName, amount: amount }) }}</h3>
-
-  <p>{{ t('The court will review the case for the full amount of {{ fullAmount }}.', { fullAmount: fullAmount }) }}</p>
+  {% if claim.response.freeMediation === FreeMediationOption.YES %}
+    <p>{{ t('We’ll ask if they want to try mediation. If they agree, we’ll contact you with an appointment.') }}</p>
+    <p>{{ t('If they don’t want to try mediation the court will review the case for the full amount of {{ amount }}', { amount: claim.totalAmountTillToday | numeral }) }}</p>
```

Comment:
Isn't this already defined in `fullAmount`?

Reply:
good spot. copied it from another file. have updated to use fullAmount